doc/install/containers.rst

```diff
@@ -16,13 +16,17 @@ ceph/ceph
   dependencies installed.
 - TODO: add vRELNUM.2.DOT so we have a container for every point release.
```

Comment:
Do we still need this statement since we have vRELNUM.Y.Z now ?

Reply:
removed!